shared/dev/dumb-sheet.render.native.js

```diff
@@ -24,8 +28,35 @@ class Render extends Component<void, Props, any> {
     }, 300)
   }
 
+  _continueTest () {
+    if (isTesting && this.state.testIndex !== -1) {
+      const {componentsOnly} = this._getComponents()
+      if (this.state.testIndex >= componentsOnly.length) {
+        this.setState({testIndex: -1})
+      } else {
+        this.setState({testIndex: this.state.testIndex + TEST_RENDER_NUMBER})
+      }
+    }
+  }
+
+  componentDidMount () {
+    if (isTesting) {
+      this._continueTest()
+    }
+  }
+
+  componentDidUpdate () {
+    setTimeout(() => {
+      this._continueTest()
+    }, 1)
+  }
+
   render () {
-    const filter = this.props.dumbFilter.toLowerCase()
+    // return isTesting ? this.renderAll() : this.renderSingle()
```

Comment:
no this was just me making this work in ci quickly. my intention is that it works 'normally' for us in the simulator but does the auto incrementing in test